### PR TITLE
Add access fund

### PR DESCRIPTION
**Pull request by eveline-xie**
Issue #70

### CODE REVIEW

File: companies/access-fund.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+blm_statements:
+  - url: https://www.instagram.com/p/CA5qXOulB1D/
+    date_posted: 2020-06-01
+  - url: https://www.accessfund.org/open-gate-blog/to-our-community
+    date_posted: 2020-06-03
+exec_team_url: https://www.accessfund.org/inside-access-fund/staff
+diversity_inclusion_url: https://www.accessfund.org/open-gate-blog/access-funds-commitment-to-diversity-equity-and-inclusion
+---
+
+Soloing is a dangerous game. One mistake—one slip-up—and you’re dead. Most climbers will never solo. Those who do have taken a very deliberate, intentional step. Once they reach the top of the wall or get back down to the ground, they can choose to never risk their life again.
```
Comment (kelseyhuse30) [inline companies/access-fund.md:13]: Awesome, thanks for transcribing!

File: companies/realtor.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (kelseyhuse30) [inline companies/realtor.md:1]: I'm not sure how this file got included :( Can you remove it from this branch?

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]

File: companies/realtor.md (hunk 2 above)
Comment (eveline-xie) [reply companies/realtor.md:1]: Oh no, can I remove it by deleting the file?

Comment (eveline-xie) [reply companies/realtor.md:1]: Screen Shot 2020-10-25 at 12 28 07 PM
Delete the file like this?

Comment (kelseyhuse30) [reply companies/realtor.md:1]: Try this instead:

In your text editor, delete the file manually.
then `git add .`
`git commit -m "remove realtor.com"
`git push`

Hopefully that should work 🤞🏻

**Comment by kelseyhuse30** [comment]
oh no lol @eveline-xie can you try resolving the merge conflict? Idk what happened.

**Review by kelseyhuse30 — APPROVED** [review]

**Pull request merged**